lib/pengine/clone.c

```diff
@@ -933,8 +933,8 @@ pe__clone_text(pcmk__output_t *out, va_list args)
         return rc;
     }
 
-    print_everything = pcmk__str_in_list(only_rsc, rsc_printable_id(rsc)) ||
-                       (strstr(rsc->id, ":") != NULL && pcmk__str_in_list(only_rsc, rsc->id));
+    print_everything = pcmk__str_in_list(only_rsc, rsc_printable_id(rsc), false) ||
+                       (strstr(rsc->id, ":") != NULL && pcmk__str_in_list(only_rsc, rsc->id, false));
```

Comment:
Here and a few below are more places that use true/false

Reply:
Fixed.